packages/babel-plugin-transform-regenerator/src/index.ts

```diff
@@ -1,3 +1,30 @@
-// eslint-disable-next-line @babel/development/plugin-name
+import { declare } from "@babel/helper-plugin-utils";
 import regeneratorTransform from "regenerator-transform";
-export default regeneratorTransform.default;
+
+export default declare(({ types: t, assertVersion }) => {
+  assertVersion(7);
+
+  return {
+    name: "transform-regenerator",
+
+    inherits: regeneratorTransform.default,
+
+    visitor: {
+      // We visit MemberExpression so that we always transform
+      // regeneratorRuntime before babel-plugin-polyfill-regenerator.
+      MemberExpression(path) {
+        const obj = path.get("object");
+        if (obj.isIdentifier({ name: "regeneratorRuntime" })) {
+          try {
+            obj.replaceWith(
+              t.callExpression(this.addHelper("regeneratorRuntime"), []),
```

Comment:
Nit: we can check `this.file.availableHelper("regeneratorRuntime")`, the replacement doesn't need special error handling.